### PR TITLE
DMOH-12: Added the OpenNow & OpenNowHTML methods to the ChannelService.

### PR DESCRIPTION
**Pull request by zero2one**
## Description

Adds methods to the ChannelService to get the Open Now status for the given Channel.

* `ChannelService::openNow` : Get the open now status as OpenNow value object.
* `ChannelService::openNowHtml` : Get the open now status as html.

## Motivation and Context

Easy access to the OpenNow API methods.

## How Has This Been Tested?

* PHPunit.
* Integration tests using examples.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have updated the CHANGELOG accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: src/ChannelService.php [hunk 1]
```diff
@@ -105,4 +107,113 @@ public function getById($serviceId, $channelId)
...
+     * @throws \StadGent\Services\OpeningHours\Exception\NotFoundException
+     * @throws \StadGent\Services\OpeningHours\Exception\ChannelNotFoundException
+     * @throws \StadGent\Services\OpeningHours\Exception\ServiceNotFoundException
+     */
+    public function openNow($serviceId, $channelId)
+    {
+        $cacheKey = $this->createChannelCacheKey(__FUNCTION__, $serviceId, $channelId);
+
+        // By default from cache.
+        $cached = $this->cacheGet($cacheKey);
```
Comment (Jelle-S) [inline src/ChannelService.php:133]: How long is this cached? The return value of open"Now" can change every minute.

File: src/Request/Channel/OpenNowHtmlRequest.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        $uri = sprintf(
+            'services/%d/channels/%d/open-now',
+            (int) $serviceId,
+            (int) $channelId
+        );
+
+        parent::__construct(
+            MethodType::GET,
+            $uri,
+            ['Accept' => AcceptType::HTML]
```
Comment (Jelle-S) [inline src/Request/Channel/OpenNowHtmlRequest.php:35]: As discussed this can be refactored to a property/constant

**Review by Jelle-S — CHANGES_REQUESTED** [review]

File: src/ChannelService.php (hunk 1 above)
Comment (zero2one) [reply src/ChannelService.php:133]: I will remove caching of the "OpenNow" response in the refactoring PR.

File: src/Request/Channel/OpenNowHtmlRequest.php (hunk 2 above)
Comment (zero2one) [reply src/Request/Channel/OpenNowHtmlRequest.php:35]: Will be fixed in the refactoring PR.

**Review by Jelle-S — APPROVED** [review]

**Pull request merged**